### PR TITLE
Array Shuffle Program

### PR DESCRIPTION
**Pull request by AmrutaDhage20**

### CODE REVIEW

File: PracticePrograms/src/FifthJune/Shuffle.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package FifthJune;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class Shuffle {
+
+    public static void shuffle(int numbers[]) {
```
Comment (waghmaresuyog) [inline PracticePrograms/src/FifthJune/Shuffle.java]: change method name

File: PracticePrograms/src/FifthJune/Shuffle.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        }
+    }
+    private static void swap_elements(int[] numbers, int index, int result) {
+        int temp = numbers[index];
+        numbers[index] = numbers[result];
+        numbers[result] = temp;
+    }
+    public static void main(String[] args) {
+        {
+            int[] numbers = {1, 2, 3, 4, 5, 6};
```
Comment (waghmaresuyog) [inline PracticePrograms/src/FifthJune/Shuffle.java:22]: give meaning name for number

File: PracticePrograms/src/FifthJune/Shuffle.java (hunk 1 above)
Comment (AmrutaDhage20) [reply PracticePrograms/src/FifthJune/Shuffle.java]: Done

File: PracticePrograms/src/FifthJune/Shuffle.java (hunk 2 above)
Comment (AmrutaDhage20) [reply PracticePrograms/src/FifthJune/Shuffle.java:22]: Done

**Review by waghmaresuyog — APPROVED** [review]

**Review by waghmaresuyog — APPROVED** [review]
Approve

**Review by amitsahu8 — APPROVED** [review]

**Pull request merged**